default/game.cfg

```diff
@@ -0,0 +1,14 @@
+# Joystick control mode. Value: 0 - off, 1 - digital, 2 - analog
+JOY_MODE 2
+
+# Joystick device number. Value: 0 - first joystick, 1 - second joystick, etc.
+JOY_NUM 0
+
+# Joystick deadzone. Value range: 0 - 65535
+JOY_DEADZONE 5000
+
+# Screen scale. Value: 1 - original, 2 - double
+SCALE 1
+
+# Display mode. Value: 0 - windowed, 1 - fullscreen
+FULLSCREEN 0
```

Comment:
There is no need to include a default game config, as the game will generate one on the fly if it doesn't find an existing one.
The same goes about the default score file.

Reply:
I had some issues with those and got Segmentation Fault on Bittboy. Not sure if it was the reason as I made other changes as well, but for completeness I have put defaults in there. Feel free to remove them.